extensions/content-usage-metrics/manifest.json

```
@@ -3083,25 +3115,28 @@
   },
   "files": {
     "app.R": {
-      "checksum": "9503ad4badfebc475f44466b7d7c8ef7"
+      "checksum": "93046fabfbe4c7e1e1360ba431583cf0"
     },
     "get_usage.R": {
-      "checksum": "5b2476731ce8e473c09abd1c593ccfa7"
+      "checksum": "f232066323b2f15d618663c2c2e97d0a"
+    },
+    "www/styles.css": {
+      "checksum": "86129d1eb71c75924c7584af191c4cd9"
     }
   },
   "users": null,
   "extension": {
-    "name": "content-usage-metrics-dashboard",
-    "title": "Content Usage Metrics Dashboard",
-    "description": "See what content is popular on your Connect instance. Dive into details for a content item to learn who's viewing it, and when.",
-    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/who-is-visiting-this-content",
-    "tags": ["R", "shiny", "usage-metrics"],
+    "name": "content-usage-metrics",
+    "title": "Content Usage Metrics",
+    "description": "understand content usage patterns on your Connect server. Browse an overview of usage across your published content, and dive into a detailed breakdown of visit-level data for each piece of content.",
```

Started with a suggestion to capitalize since descriptions are mostly using sentence structure, but did some word smithing to avoid the word "Content" if we wanted to do that. Feel free to take or leave parts of this.

```suggestion
    "description": "Understand usage patterns of your apps on your Connect server. Browse an overview of usage across everything you've published, and dive into detailed breakdowns of data for each visit.",
```

---

Thanks for catching the capitalization error! Gonna make a few small edits of your edits and commit. :)

This also makes me think: In the interest of avoiding the word "Content", perhaps `content-usage-metrics` is not the best name for this. Gonna post to Slack for other ideas.

---

- Trying to avoid two successive "your"s. I feel like we could say "on _this_ Connect server" as this is being displayed in the Gallery, but not sure.
- To me, "data for each visit" seems to miss the point: that they can view the level (1) on an overview level and (2) on an item-specific level, like, an individual piece of content.

```suggestion
    "description": "Understand usage patterns across your work on Connect. Browse an overview of usage across everything you've published, and dive into detailed breakdowns of data for an individual deployment, down to the level of individual visits.",
```

---

Agreed your final description is much better.